core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_pluginconfig.yaml

```diff
@@ -0,0 +1,28 @@
+approve:
+- commandHelpLink: ""
+  ignore_review_state: true
+  repos:
+  - openshift/coredns-ocp-dnsnameresolver
+  require_self_approval: true
+label:
+  restricted_labels:
+    openshift/coredns-ocp-dnsnameresolver:
+    - allowed_users:
+      - candita
+      - Miciah
+      label: backport-risk-assessed
+    - allowed_users:
+      - lihongan
+      - quarterpin
```

Comment:
`quarterpin` you can remove this one, as he is no more with the organization

Reply:
Thanks @melvinjoseph86 . Will update it. Should there be anybody else who needs to be added here?

Reply:
We are only three now...